docs(slideshow): doc text update (#187)
lumapps/design-system: demo/react/doc/product/components/slideshow.mdx

import {
    ImageBlock,
    ImageBlockCaptionPosition,
    ImageBlockProps,
    Slideshow,
    SlideshowItem,
    ThumbnailAspectRatio,
} from 'LumX';

# Slideshow

> **Slideshows display multiple pieces of featured content on a limited allocated amount of space.**

Reviewer: ```suggestion
**Slideshows display multiple pieces of featured content in a limited amount of space.**
```
